main/acle.md

```diff
@@ -4,7 +4,7 @@ version: 2024Q2
 date-of-issue: 21 June 2024
```

Comment:
is this date correct?

Reply:
This will be changed to the Q3 release date